lxd/storage/drivers/volume_interface.go

```diff
@@ -0,0 +1,108 @@
+package drivers
+
+import (
+	"github.com/lxc/lxd/lxd/locking"
+	"github.com/lxc/lxd/lxd/operations"
+)
+
+// VolumeInterface is an interface for drivers.Volume.
+type VolumeInterface interface {
```

Comment:
You define this interface but then never use it anywhere so it can probably go away :)

I see you use a `volume` interface in the lifecycle package which uses a subset of this, but it's not depend on this interface existing, so I think it's fine to just drop it.

Reply:
Sounds good, it's gone :)